Round 2 of modeling

dustinpierce opened the pull request:
This work contains our most current version of modeling, visualizations and text. Please review and let me know if any changes need to be made. Thanks.

dustinpierce committed:
Update stroke_data_eda.Rmd

Round 2 of modeling -- interim save

97caguilar reviewed (approved):
Just needs a brief description of the PCA chunk--otherwise good to go.

dustinpierce committed:
Update stroke_data_eda.Rmd

Final pass through notebook -- cleaning up code, removing all non-essential information and adding assorted details (labels, etc.) as needed

dustinpierce committed:
Update stroke_data_eda.Rmd

Notebook complete

dustinpierce commented:
Merging this, notebook is complete
